collections/_article/godot-game-awards-stayathomedev.md

```diff
@@ -0,0 +1,57 @@
+
+### What inspired you to create the Godot Game Awards, and how long has it been running?
+I started the “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series over two years ago on my channel.  At first it started as a compilation of cool projects or games I found throughout each week.  About 7-8 videos in, I thought it would be cool if people could vote for their favorite each week and the engagement exploded.  People would comment every week and eventually I compiled every winner into a final vote for the “[Best Godot Project of the Year](https://www.youtube.com/watch?v=3_3N_nL9gIM)” in 2023.  This year I wanted to expand that into multiple categories, highlighting certain achievements like art design and music, and give attention to other things like plugins, tutorials, even devlogs.  From that came the Godot Game Awards, a compilation of the best Godot-based games, plugins, and projects as voted by viewers on my channel.
+
+
+### How can people get involved in this year’s edition?
+People can get involved from now until December 15th by voting on the 20+ different categories with an official 2024 🗳️ [Godot Game Awards Ballot](https://forms.gle/Q4yYy82neZcKA3Ny6).
+
+The winners will be announced in a premiere video on December 28th at 2:00pm EST on my channel so that everyone can watch together and help support the great work by these developers.
+
+
+### Tell us about your YouTube channel—what kind of content do you focus on, and what motivated you to start creating videos?
+I started [StayAtHomeDev](https://www.youtube.com/@stayathomedev) over two years ago to share my passion for creating through game development.  At the time, I had a potential job offer to be the Web Coordinator for a professional football team in the US.  But I had just been given a grant by Meta for some VR development work with their then Oculus Quest 2 headset so I used that to launch the channel. It really was an exercise to create independent income so I could stay at home with my kids.  That was a lifestyle choice I knew I wanted to make for them, so I took a shot at Youtube.
+
+I’m very interested in sharing knowledge so the channel has had an element of teaching with tutorials or game development concept videos.  With the “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, the channel started to steer more towards enriching the Godot community and showcasing what people were creating with the engine.  Now, it’s a mix of tutorials, QAs about general game development topics, and the newest format, interviews.  I am actually most excited about the interview videos because they really give a chance for viewers to peek into real game dev life with stories and advice from developers actively breaking through the industry or those who are already successful, and of course doing it with the Godot Engine.
```

Comment:
```suggestion
I’m very interested in sharing knowledge so the channel has had an element of teaching with tutorials or game development concept videos.  With the “[This Week in Godot](https://www.youtube.com/playlist?list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, the channel started to steer more towards enriching the Godot community and showcasing what people were creating with the engine.  Now, it’s a mix of tutorials, QAs about general game development topics, and the newest format, interviews.  I am actually most excited about the interview videos because they really give a chance for viewers to peek into real game dev life with stories and advice from developers actively breaking through the industry or those who are already successful, and of course doing it with the Godot Engine.
```

Actually, by this point maybe we don't need to keep linking the series, so maybe just this instead:

```suggestion
I’m very interested in sharing knowledge so the channel has had an element of teaching with tutorials or game development concept videos.  With the “This Week in Godot” series, the channel started to steer more towards enriching the Godot community and showcasing what people were creating with the engine.  Now, it’s a mix of tutorials, QAs about general game development topics, and the newest format, interviews.  I am actually most excited about the interview videos because they really give a chance for viewers to peek into real game dev life with stories and advice from developers actively breaking through the industry or those who are already successful, and of course doing it with the Godot Engine.
```